### PR TITLE
Initial Junos EVO support

### PR DESCRIPTION
**Pull request by claudiolor**
## Description

This PR introduces initial support for Junos EVO devices. The following services are available:

- aprnd
- bgp
- devconfig
- device
- evpnVni
- interfaces
- inventory
- lldp
- macs
- ospfIf
- ospfNbr
- routes
- vlan

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: suzieq/poller/worker/nodes/node.py [hunk 1]
```diff
@@ -1892,6 +1902,10 @@ async def _parse_init_dev_data_devtype(self, output, cb_token) -> None:
             data = output[1]["data"]
             hmatch = re.search(r'\nHostname:\s+(\S+)\n', data)
             if hmatch:
+                # TODO
```
Comment (claudiolor) [inline suzieq/poller/worker/nodes/node.py:1905]: Should we need to address this TODO now?

File: suzieq/poller/worker/nodes/node.py [hunk 2]
```diff
@@ -1863,18 +1868,23 @@ async def _rest_gather(self, service_callback, cmd_list, cb_token,
...
-        await self._exec_cmd(self._parse_init_dev_data,
-                             ["show system uptime|display json",
-                              "show version"], None, 'mixed',
-                             reconnect=reconnect)
+        if self.devtype == 'junos-evo':
+            cmdlist = ["show system uptime|display json",
+                       'show version node all']
+        else:
+            cmdlist = ["show system uptime|display json",
+                       'show version']
```
Comment (LucaNicosia) [inline suzieq/poller/worker/nodes/node.py:1876]: @ddutt why don't create a new class for junos evo instad of doing this if else? It seems a cleaner solution

Comment (ddutt) [reply suzieq/poller/worker/nodes/node.py:1876]: Because we'll be duplicating a lot of work

File: suzieq/poller/worker/nodes/node.py (hunk 1 above)
Comment (ddutt) [reply suzieq/poller/worker/nodes/node.py:1905]: Not right now.

**Review by ddutt — APPROVED** [review]

**Review by ddutt — APPROVED** [review]

**Pull request merged**